etna/transforms/timestamp/holiday.py

```diff
@@ -1,20 +1,51 @@
-import datetime
 from enum import Enum
 from typing import List
 from typing import Optional
 
 import holidays
 import numpy as np
 import pandas as pd
-
+from pandas.tseries.offsets import MonthBegin
+from pandas.tseries.offsets import MonthEnd
+from pandas.tseries.offsets import QuarterBegin
+from pandas.tseries.offsets import QuarterEnd
+from pandas.tseries.offsets import Week
+from pandas.tseries.offsets import YearBegin
+from pandas.tseries.offsets import YearEnd
+
+from etna.datasets import TSDataset
 from etna.transforms.base import IrreversibleTransform
 
 
+def bigger_than_day(freq: str):
+    """Compare frequency with day."""
+    dt = "2000-01-01"
+    dates_day = pd.date_range(start=dt, periods=2, freq="D")
+    dates_freq = pd.date_range(start=dt, periods=2, freq=freq)
+    return dates_freq[-1] > dates_day[-1]
+
+
+def define_period(offset: pd.tseries.offsets.BaseOffset, dt: pd.Timestamp, freq: str):
```

Comment:
I'm also not sure that we need `freq` here. Because `offset` has `freqstr` parameter. But it could be different from `freq` which could be a problem for usability.

Reply:
we only need freq to show original frequency of user dataset in ValueError.